help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Snap Inc
+description: Use your audience segments in the Snapchat Ads Platform.
+---
+
+# Snap Inc
+
+## Overview {#overview}
+
+[Snapchat Ads](https://forbusiness.snapchat.com/) are made for every business, no matter the size or industry. Become a part of Snapchatters' everyday conversations with full-screen, digital ads that inspire action from the people that matter most to your business.
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *Snap Inc* team. For any inquiries or update requests, please contact them directly at *dev-support@snap.com*
+
+## Use cases {#use-cases}
+
+This destination allows marketers to import user segments that they built using the Adobe platform into Snapchat Ads and use them to target their ads. 
```

Comment:
Minor: 

import user segments that they built using the Adobe platform [...] ->  import user segments created in Experience Platform [...]